docs/content/installation/install_driver_openshift_web.md

```diff
@@ -18,7 +18,7 @@ The Red Hat OpenShift Container Platform uses the following `SecurityContextCons
 
 3. Search for IBM block storage CSI driver.
```

Comment:
I can't comment on the third line of this file, but what about renaming there?

Reply:
again, this isn't writing the actual _name_ of the operator but just referring to it. no need to update.
keeping as-is

> When using the Red Hat® OpenShift® Container Platform, the operator for IBM® block storage CSI driver can be installed directly from OpenShift web console, through the 

OperatorHub.